lib/ansible/modules/network/nxos/nxos_igmp_interface.py

```diff
@@ -115,85 +118,70 @@
               device does not send group-specific queries.
               The default is disabled.
         type: bool
-        default: 'no'
     oif_routemap:
         description:
-            - Configure a routemap for static outgoing interface (OIF).
+            - Configure a routemap for static outgoing interface (OIF) or
+              keyword 'default'.
     oif_prefix:
         description:
-            - Configure a prefix for static outgoing interface (OIF).
+            - Configure a prefix for static outgoing interface (OIF). This
+              argument is deprecated, please use oif_ps instead.
     oif_source:
         description:
-            - Configure a source for static outgoing interface (OIF).
+            - Configure a source for static outgoing interface (OIF). This
```

Comment:
Put the deprecation notice first:
```
- This argument is deprecated.... 
- Configure a source... 
```

By the time Ansible 2.6 is released we may have a better way of displaying deprecation notices for modules, if so we will go back and update all of these.